### ISSUE TITLE
Changing text in normal mode commands in R buffer not working

### ISSUE THREAD

**Issue by oli666** [jalvesaq/Nvim-R#458]
I am trying to do the following: 
- Starting R using \\rf
- Moving the cursor to the R buffer split window
- Typing "mtcars %>% filter(cyl == 4)"
- Using ESC to get to normal mode
- Hitting "F4" to positon cursoer at "4"
- Trying to use "r6" to change "4" to "6" 

At the last step, I get the error message "E21: cannot make changes, 'modifiable' is off"

I have also tried to ":set modifiable" for the buffer, but this does not have any effect.

**Comment by jalvesaq** [comment]
I agree that commands that were not sent to the interpreter yet should be editable in Normal mode, but neither Vim nor Neovim can do this.

Anyway, this is not a Nvim-R issue, and you should report it to either Vim or Neovim developers. They can replicate the issue with the following procedure:

1. Start a terminal with the command `:terminal`.
2. Type in Terminal mode:
```
ls -a
```
3. Press <kbd>Ctrl</kbd><kbd>\\</kbd> <kbd>Ctrl</kbd><kbd>n</kbd> to go to Normal mode.
4. Put the cursor over the letter "a" and press <kbd>r</kbd><kbd>l</kbd>.

**Comment by inventionate** [comment]
Just a little hint / idea. Maybe it would be an option to use prompt buffers instead of terminal windows:

https://vimhelp.org/channel.txt.html#prompt-buffer

https://github.com/neovim/neovim/issues/10431
